cmd/download.go

```diff
@@ -8,14 +8,18 @@ import (
 	"github.com/spf13/cobra"
 )
 
+func init() {
```

Comment:
This is invalid syntax. Looks like the merge got messed up.
Note: instead of merging you should have *rebased* onto the latest version of main, this would have created a much better history.
But that's okay, we will fix this once we squash this into just one commit.